nipype/interfaces/ants/segmentation.py

```diff
@@ -335,7 +335,6 @@ class N4BiasFieldCorrectionInputSpec(ANTSCommandInputSpec):
     )
     rescale_intensities = traits.Bool(
         False,
```

Comment:
There was a move a while back to either remove defaults if unused, or set `usedefault=True` if the default was specified. In this case, if we're removing `usedefault`, we should also remove the default.

```suggestion
```